.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```

Comment:
The "clear and concise" bit here can be removed and perhaps changed to mention that this section is optional, I'd imagine it's fine to report an issue which you're not entirely clear how it'll be solved yet.

Reply:
We can vary the wording, of course. But the value here, in my opinion, is accurately capturing as two distinct entities what the problem is, in of itself, vs what the author believes the solution is. Often these two are confused and/or transposed..